cpp/oneapi/dal/algo/rbf_kernel/backend/gpu/compute_kernel_dense_dpc.cpp

```diff
@@ -29,46 +27,112 @@ using input_t = compute_input<task::compute>;
 using result_t = compute_result<task::compute>;
 using descriptor_t = detail::descriptor_base<task::compute>;
 
-namespace daal_rbf_kernel = daal::algorithms::kernel_function::rbf;
-namespace interop = dal::backend::interop;
+namespace pr = dal::backend::primitives;
+
+template <typename Float>
+inline auto compute_rbf_values(sycl::queue& queue,
```

Comment:
You have two function, `compute_rbf` and `compute_rbf_values`, difference is not clear. Rename this function to `compute_exponents`?